scripts/parser-tests/bump-test262-version.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+# read given commit SHA1 from stdin, update to TEST262_COMMIT in Makefile
+# usage:
+# sh ./scripts/parser-tests/get-test262-version.sh | sh ./scripts/parser-tests/bump-test262-version.sh
+
+set -e
+perl -i -pe 's/^TEST262_COMMIT.+$/TEST262_COMMIT = '$(cat)'/' ./Makefile
```

Comment:
I use `perl` because it passes on both macOS and Linux while the `sed` on FreeBSD is different with the GNU one.